tests/libs/datasets/sources/cdc_testing_dataset_test.py

```diff
@@ -23,7 +23,7 @@ def test_cdc_testing_modify_dataset():
     ds_out = cdc_testing_dataset.modify_dataset(ds_in)
 
     dc_data_after_filter = {
-        CommonFields.TEST_POSITIVITY_7D: [0.5, 0.6, None, None, None, None],
+        CommonFields.TEST_POSITIVITY_7D: [0.5, 0.6, 0, 0, 0, 0],
```

Comment:
This test looks kind of silly now, since it used to be testing that trailing zeros are removed (among other things I guess) but now it's just got a bunch of trailing zeros for no reason, I guess.

Maybe we can put this test back if we make my other suggested changes?  Else perhaps we should either modify this test to remove the trailing zeros or add a comment explaining that we used to test removal of trailing zeros but no longer are doing that. 🤷